param/parameterized.py

```diff
@@ -2815,6 +3178,9 @@ def values(self_, onlychanged=False):
         vals.sort(key=itemgetter(0))
         return dict(vals)
 
+    # Please update the docstring with better description and examples
+    # I've (MarcSkovMadsen) not been able to understand this. Its probably because I lack context.
+    # Its not mentioned in the documentation or pytests
     def force_new_dynamic_value(self_, name): # pylint: disable-msg=E0213
```

Comment:
I hope someone else than me will update this docstring. I lack context and cannot find it in the documentation or pytests.